WordPress/wordcamp.org
# Add new default Jetpack modules

In #494 it has been discussed what new Jetpack modules should be enabled by default when creating new WordCamp sites.

Based on that discussion, this PR added the following modules to the list of default enabled ones. At the same time, did minor improvements to the readability of the `default_jetpack_modules` function.

* Sharing (`sharedaddy`) - needed for og: tags and social shares looking nice, usually forgotten to put on by organisers
* Copy Post - useful little tool on the admin side, no effect on performace
* Sitemap - doesn't hurt on SEO side
* Site Accelerator (`photon` and `photon-cdn`) - I don't see any reasons why not to make use of someone else's server resources and save ours 😅

Fixes #494 

Props @iandunn, @coreymckrill, @andreamiddleton

### How to test the changes in this Pull Request:

1. Create a new site
2. In Jetpack settings, check that mentioned modules are enabled
